src/startup-tasks/task-start-consumers.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  name: 'Start event consumers',
```

Comment:
If that's what you did in `windbreaker-hooks` I'll make the switch so that we can stay consistent.